src/main/java/org/intellij/sdk/codesync/ProjectOpenCloseListener.java

```diff
@@ -39,6 +43,9 @@ public void projectOpened(@NotNull Project project) {
       return;
     }
 
+    // Schedule buffer handler.
```

Comment:
Maybe call first time directly and the schedule for every 5s?

Reply:
Any reason for this? I mean scheduling also calls it pretty much right away.

Reply:
Then we are good. I thought it will run first time after 5s.